docker-entrypoint.sh

```diff
@@ -8,8 +8,6 @@ set -o pipefail
 
 if [ "$#" -eq 0 ]; then
   # Print select tool versions then open an bash shell
-  echo -en "aws-cli\t\t" && command aws --version | awk -F' ' '{print $1}' | awk -F'/' '{print $2}'
```

Comment:
Any reason not to split out multiple entrypoint scripts to preserve the version output of aws-cli and azure-cli for the images where it is appropriate?

Reply:
Not a bad idea but another reason I removed them was because they are slow